src/main/java/randoop/main/Minimize.java

```diff
@@ -0,0 +1,1078 @@
+        return new Outputs("", "I/O error occurred when running process.", 1);
+      }
+    } else {
+      // Input Java file is in a package, execute in the root directory.
+      try {
+        process = Runtime.getRuntime().exec(command, null, new File(executionDir));
+      } catch (IOException e) {
+        return new Outputs("", "I/O error occurred when running process.", 1);
+      }
+    }
+
+    final TimeLimitProcess timeLimitProcess = new TimeLimitProcess(process, timeoutLimit * 1000);
+
+    Callable<String> stdCallable =
+        new Callable<String>() {
+          @Override
+          public String call() throws Exception {
+            try {
+              return IOUtils.toString(timeLimitProcess.getInputStream(), Charset.defaultCharset());
+            } catch (IOException e) {
+              // Error reading from process's input stream.
+              return "Error reading from process's input stream.";
+            }
+          }
+        };
+
+    Callable<String> errCallable =
+        new Callable<String>() {
+          @Override
+          public String call() throws Exception {
+            try {
+              return IOUtils.toString(timeLimitProcess.getErrorStream(), Charset.defaultCharset());
+            } catch (IOException e) {
+              // Error reading from process's error stream.
+              return "Error reading from process's error stream.";
+            }
+          }
+        };
+
+    ExecutorService fixedThreadPool = Executors.newFixedThreadPool(2);
+    Future<String> stdOutput = fixedThreadPool.submit(stdCallable);
+    Future<String> errOutput = fixedThreadPool.submit(errCallable);
+    fixedThreadPool.shutdown();
+
+    // Wait for the TimeLimitProcess to finish.
+    try {
+      timeLimitProcess.waitFor();
+    } catch (InterruptedException e) {
+      return new Outputs("", "Process was interrupted while waiting.", 1);
+    }
+    if (timeLimitProcess.timed_out()) {
+      timeLimitProcess.destroy();
+      return new Outputs("", "Process timed out after " + timeoutLimit * 1000 + " seconds.", 1);
+    }
+
+    String stdOutputString;
+    String errOutputString;
+    try {
+      stdOutputString = stdOutput.get();
+      errOutputString = errOutput.get();
+    } catch (InterruptedException e) {
+      return new Outputs("", "Process was interrupted while waiting.", 1);
+    } catch (ExecutionException e) {
+      return new Outputs("", "A computation in the process threw an exception.", 1);
+    }
+
+    // Collect and return the results from the standard output and error output.
+    return new Outputs(stdOutputString, errOutputString, timeLimitProcess.exitValue());
+  }
+
+  /**
+   * Normalize the standard output obtained from running a JUnit test suite. By normalizing the
+   * {@code String} representation of the output, we remove any extraneous information such as line
+   * numbers. The relevant information that is retained includes the indices of each failing test
```

Comment:
Previously "handled" line numbers by ignoring all lines with line numbers. The minimizer now uses the whole stack trace and removes the line numbers from the trace.